<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-20</y>
     <width>1420</width>
     <height>920</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.0511364, y1:0.0687727, x2:1, y2:1, stop:0.363636 rgba(5, 82, 201, 255), stop:0.9375 rgba(0, 0, 0, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 75 36pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>651</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Sign in to your account </string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>730</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_textbx">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_textbx">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Username :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_lbl">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>600</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>username_textbx</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>login</zorder>
   <zorder>password_textbx</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>error_lbl</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
